### DISCUSSION TITLE
Tracking secret habits

### DISCUSSION THREAD

**Discussion by Topherfx** [iSoron/uhabits#1262] [Feature Requests] [answered]
Hi,
would it be possible to add the option to hide specific habbits from the overview (maybe even with a Pin). I use the app to track many things, including taking my meds. Sometimes I do compare some items with friends as we habe the same goals. But the don't need to see the really personal stuff.
Keep up the great work!
Topher

**Comment by hiqua** [comment]
What about renaming them to something only you know?

**Comment by iSoron** [comment, marked as answer]
Duplicate of #387
